editor/src/core/shared/github.ts

```diff
@@ -374,7 +382,14 @@ export async function updatePullRequestsForBranch(
         break
       case 'SUCCESS':
         dispatch(
-          [updateGithubData({ currentBranchPullRequests: responseBody.pullRequests })],
+          [
+            updateGithubData({
+              currentBranchPullRequests: responseBody.pullRequests.map((pr) => ({
+                ...pr,
+                number: getPullRequestNumberFromUrl(pr.htmlURL),
```

Comment:
There is a ` number` field in the response Github gives us that we can use for this.

Reply:
https://github.com/concrete-utopia/utopia/pull/2827/commits/9392aa53365b8455d504b6fc9b578fd0d22c9215